src/opnsense/mvc/app/controllers/OPNsense/Proxy/forms/main.xml

```diff
@@ -239,10 +229,9 @@
                 <help><![CDATA[
                   Enable SSL inspection mode,
                   which allows to log HTTPS connections information, such as requested URL and/or
-                  make the proxy act as a man in the middle between the internet and your clients.<br/>
+                  make the proxy act as a man in the middle between the internet and your clients.
                   Be aware of the security implications before enabling this option.
-                  <br/><br/>
-                  If you plan to use transparent HTTPS mode, you need nat rules to reflect your traffic.<br/>
+                  If you plan to use transparent HTTPS mode, you need nat rules to reflect your traffic.
                   <a href="/firewall_nat_edit.php?template=transparent_proxy&https=1">Add a new firewall rule </a>
```

Comment:
How about this hypertext? This has a function now (linking to the template for automatic adding of Port forward. This will loose it's functionality when rigidly applying the development direction for XML text strings?